Error importing unittest: No module named xmlrunner

If I replay the [tutorial](http://pybuilder.github.io/documentation/tutorial.html) I get an error when I try to execute the first pyb with unit testing.

I attach the complete session (the full error is in the last lines):

	$ virtualenv venv
	New python executable in venv/bin/python
	Installing setuptools, pip, wheel...done.
	$ source venv/bin/activate
	(venv)$ pip install pybuilder
	Collecting pybuilder
	Collecting tblib (from pybuilder)
	Collecting six (from tblib->pybuilder)
	  Using cached six-1.10.0-py2.py3-none-any.whl
	Installing collected packages: six, tblib, pybuilder
	Successfully installed pybuilder-0.11.2 six-1.10.0 tblib-1.1.0
	(venv)$ mkdir src/main/python
	mkdir: src/main: No such file or directory
	(venv)$ mkdir -p src/main/python
	(venv)$ vim !$/helloworld.py
	from mockito import mock, verify
	vim src/main/python/helloworld.py
	from pybuilder.core import init, use_plugin
	(venv)$ vim build.py
	(venv)$ pyb
	PyBuilder version 0.11.2
	Build started at 2015-11-23 17:57:26
	------------------------------------------------------------
	[INFO]  Building test_pyb version 1.0.dev0
	[INFO]  Executing build in /data/home/fb/torkspace/test_pyb
	[INFO]  Going to execute task publish
	[INFO]  Building distribution in /data/home/fb/torkspace/test_pyb/target/dist/test_pyb-1.0.dev0
	[INFO]  Copying scripts to /data/home/fb/torkspace/test_pyb/target/dist/test_pyb-1.0.dev0/scripts
	------------------------------------------------------------
	BUILD SUCCESSFUL
	------------------------------------------------------------
	Build Summary
	             Project: test_pyb
	             Version: 1.0.dev0
	      Base directory: /data/home/fb/torkspace/test_pyb
	        Environments:
	               Tasks: prepare [0 ms] compile_sources [0 ms] run_unit_tests [0 ms] package [0 ms] run_integration_tests [0 ms] verify [0 ms] publish [0 ms]
	Build finished at 2015-11-23 17:57:26
	Build took 0 seconds (3 ms)
	(venv)$ mkdir src/main/scripts
	from pybuilder.core import init, use_plugin
	(venv)$ vim src/main/scripts/hello-pybuilder
	(venv)$ pyb
	PyBuilder version 0.11.2
	Build started at 2015-11-23 17:58:23
	------------------------------------------------------------
	[INFO]  Building test_pyb version 1.0.dev0
	[INFO]  Executing build in /data/home/fb/torkspace/test_pyb
	[INFO]  Going to execute task publish
	[INFO]  Building distribution in /data/home/fb/torkspace/test_pyb/target/dist/test_pyb-1.0.dev0
	[INFO]  Copying scripts to /data/home/fb/torkspace/test_pyb/target/dist/test_pyb-1.0.dev0/scripts
	------------------------------------------------------------
	BUILD SUCCESSFUL
	------------------------------------------------------------
	Build Summary
	             Project: test_pyb
	             Version: 1.0.dev0
	      Base directory: /data/home/fb/torkspace/test_pyb
	        Environments:
	               Tasks: prepare [0 ms] compile_sources [0 ms] run_unit_tests [0 ms] package [1 ms] run_integration_tests [0 ms] verify [0 ms] publish [0 ms]
	Build finished at 2015-11-23 17:58:23
	Build took 0 seconds (3 ms)
	(venv)$ mkdir -p src/unittest/python/
	(venv)$ vim src/unittest/python/helloworld_tests.py
	(venv)$ vim build.py
	(venv)$ pyb install_dependencies
	PyBuilder version 0.11.2
	Build started at 2015-11-23 17:59:01
	------------------------------------------------------------
	[INFO]  Building test_pyb version 1.0.dev0
	[INFO]  Executing build in /data/home/fb/torkspace/test_pyb
	[INFO]  Going to execute task install_dependencies
	[INFO]  Installing all dependencies
	[INFO]  Installing build dependencies
	[INFO]  Installing dependency 'mockito'
	[INFO]  Installing runtime dependencies
	------------------------------------------------------------
	BUILD SUCCESSFUL
	------------------------------------------------------------
	Build Summary
	             Project: test_pyb
	             Version: 1.0.dev0
	      Base directory: /data/home/fb/torkspace/test_pyb
	        Environments:
	               Tasks: install_dependencies [2575 ms]
	Build finished at 2015-11-23 17:59:04
	Build took 2 seconds (2578 ms)
	(venv)$ vim build.py
	(venv)$ pyb
	PyBuilder version 0.11.2
	Build started at 2015-11-23 17:59:25
	------------------------------------------------------------
	[INFO]  Building test_pyb version 1.0.dev0
	[INFO]  Executing build in /data/home/fb/torkspace/test_pyb
	[INFO]  Going to execute task publish
	[INFO]  Running unit tests
	[INFO]  Executing unit tests from Python modules in /data/home/fb/torkspace/test_pyb/src/unittest/python
	[ERROR] Import error in test file /data/home/fb/torkspace/test_pyb/venv/lib/python2.7/site-packages/pybuilder/plugins/python/unittest_plugin.py, due to statement 'lambda stream: __import__("xmlrunner").XMLTestRunner(output=project.expand_path("$dir_target/reports"),' on line 49
	[ERROR] Error importing unittest: No module named xmlrunner
	------------------------------------------------------------
	BUILD FAILED - Unable to execute unit tests.
	------------------------------------------------------------
	Build finished at 2015-11-23 17:59:26
	Build took 0 seconds (242 ms)
	(venv)$

--- a/src/main/python/pybuilder/plugins/python/unittest_plugin.py
+++ b/src/main/python/pybuilder/plugins/python/unittest_plugin.py
@@ -39,7 +39,7 @@ use_plugin("python.core")
 
 @init
 def init_test_source_directory(project):
-    project.build_depends_on("unittest-xml-reporting")
+    project.depends_on("unittest-xml-reporting")
 
     project.set_property_if_unset("dir_source_unittest_python", "src/unittest/python")
     project.set_property_if_unset("unittest_module_glob", "*_tests")

